docs/HowTo/Run-Docker-Image.md

```diff
@@ -0,0 +1,76 @@
+description: Run Orion using the official docker image
+<!--- END of page meta data -->
+
+# Running the Orion Docker Image
+
+Use the Orion Docker image to run a single Orion node without installing Orion.
+
+__Prerequisites__:
+
+* [Docker](https://docs.docker.com/install/)
+
+In this example, the [password file](#1-create-password-file), [public/private keypair](#2-generate-keys), and
+[Orion configuration file](#3-create-a-configuration-file) are created in the
+`/Users/user1/Node1/Orion` directory.
+
+!!! important
+    Do not mount a volume at the (`/opt/orion`) path, this prevents
+    Orion from safely launching.
+
+    In this example the required volume is mounted at the `/data` path.
+
+## 1. Create Password File
+
+Create a file containing the password used to encrypt the key pair.
+
+Specify the password when [generating the keys](#2-generate-keys), the
+file is specified for the `passwords` property in the [configuration file](#3-create-a-configuration-file).
+
+## 2. Generate Keys
+
+Generate a public/private keypair for the Orion node using Docker:
+
+``` bash
+docker run -i --rm --mount type=bind,source=/Users/user1/Node1/Orion,target=/data pegasyseng/orion:develop -g /data/nodeKey
```

Comment:
I got this error 
[ERROR] [main] 12-23-2019 01:47:42 n.c.o.c.Orion - /data/nodeKey.pub
java.nio.file.AccessDeniedException: /data/nodeKey.pub

Reply:
can we do this with /data ? Is it the extra security in catalina that's making it not work for me?

Reply:
@macfarla <user1> is meant to be your username. You can always add an additional mount directory in docker settings and then update mount source command to that folder. Mac Catalina is more restricted than previous Mac versions so it is possible that Docker might trip. 

<img width="502" alt="Screen Shot 2019-12-23 at 12 56 15 pm" src="https://user-images.githubusercontent.com/669527/71332883-aeb0fe80-2583-11ea-8103-a3f01e86fafc.png">

Reply:
The mount command is meant to do:  container's "/data" -> host's "/Users/username/Node1/Orion....."